packages/mesa/build.sh

```diff
@@ -18,19 +19,30 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -Dgbm=enabled
 -Dopengl=true
 -Degl=enabled
--Degl-native-platform=x11
 -Dgles1=disabled
 -Dgles2=enabled
 -Ddri3=enabled
 -Dglx=dri
 -Dllvm=enabled
 -Dshared-llvm=disabled
--Dplatforms=x11
--Dgallium-drivers=swrast
--Dvulkan-drivers=
+-Dplatforms=x11,wayland
+-Dgallium-drivers=swrast,zink
+-Dvulkan-drivers=swrast
 -Dosmesa=true
 -Dglvnd=true
+-Dshared-glapi=enabled
+-Dshader-cache=enabled
 "
+#-Degl-native-platform=x11
```

Comment:
I don't think there is any need to remove this argument. Of course, the building system will pick up the first value of `platforms` as `egl-native-platform`, but removing this will make it ambiguous for users to know which is the default EGL platform. Besides, almost every Linux distribution uses X11.